src/components/document/student-group-view.scss

```
@@ -92,3 +92,9 @@
     }
   }
 }
+
+.shrink-student-view{
+  width: calc(100% - 36%);
```

I think this should work:
```suggestion
  width: 64%;
```

---

Ah okay, I just pushed up another version with a new RGB value so it should pass now and I tested it locally, but would you like me to test write a test to check for class instead?